### ISSUE TITLE
Put into issues

### ISSUE THREAD

**Issue by agstephens** [ncasuk/ncas-isc#56]
[ ] SJ/AI/AS/RS: 
Set up test session for training accounts
[ ] AS: contact Alison P for above
[ ] AS: Create a script to:
Generate local conda env for Notebook service
Clone the ISC course
[ ] RS/AI: Review/update basic python materials:
presentations
exercises
[ ] AS: Look at python libraries materials and work out what to change
[ ] SP: Review/update shell materials

**Comment by spepler** [comment]
Reviewed half of the materials. Looks good so far. I need to make a command line version of the exercises for use on the course.

**Comment by spepler** [comment]
I have looked at making just html pages for the handouts. This works well but how to host them?

**Comment by agstephens** [comment]
@spepler do you mean _how can we make the pages visible via github_ (because it shows the source rather than the rendered pages)?

I would rather use Markdown so that Github can render it for you. Like we do for the workshop:

https://github.com/cedadev/jasmin-workshop#readme

**Comment by spepler** [comment]
@agstephens Yes, I'm. exploring that option too. It could use cool "click here to show the solution" things with html, but md keeps it all together which I think is more  important.

**Comment by spepler** [comment]
https://github.com/ncasuk/ncas-isc/blob/master/shell/exercises/shell_exercise1_files_sol.md

**Comment by agstephens** [comment]
@spepler Looks good to me.

**Comment by spepler** [comment]
All looks good now. I have updated the exercises now.
